### ISSUE TITLE
omit MARC subfield $1 from all mappings

### ISSUE THREAD

**Issue by flapka** [ycba-cia/blacklight-collections2#472]
I notice today that URLs in MARC subfield $1 are inadvertently mapping and displaying in BL. See for example this record:
https://collections.britishart.yale.edu/catalog/orbis:3555253

There are OCLC.org urls after creator and contributor names. 

We want to treat subfield $1 in the same manner as subfield $0. Generally omit. 

The easiest fix is probably to batch replace all instances of `@code!='0'` (in the XSLT) to `@code!='0' and @code!='e1`

Thanks

**Comment by yulgit1** [comment]
All set:

suppress 100,110,111,700,710,711,720 subfield 1
https://git.yale.edu/ermadmix/ycba_xslts/commit/04557ddfefbcc840898e2e2aee9863bb3c5190d9

To index starting tonight.

**Comment by flapka** [comment]
Yes, looks fixed. Thanks for the quick change. I'll let you know if I see any other issues.
